src/main/java/com/joyent/manta/cosbench/MantaStorage.java

```diff
@@ -241,9 +265,102 @@ public void createObject(final String container,
         }
     }
 
+    /**
+     * Method for parsing and uploading to use multipart upload.
+     * @param data - the input stream
+     * @param path - the path of
+     * @throws IOException - this has the possiblity of throwing a IOException
+     */
+    private void multipartUpload(final InputStream data, final String path) throws IOException {
+        logger.debug("Client is not encrypted!");
+        ServerSideMultipartManager multipartManager = new ServerSideMultipartManager(client);
+        if (splitSize == null) {
+            splitSize = new Integer(DEFAULT_SPLIT);
```

Comment:
This seems like it belongs in `init(Config, Logger)`, any reason it isn't initialized sooner?